docs/src/fts.md

```diff
@@ -0,0 +1,48 @@
+# [EXPERIMENTAL] Full text search
+
+LanceDB now provides experimental support for full text search.
+
+## Installation
+
+To use full text search, you must install the fts optional dependencies:
+
+`pip install lancedb[fts]`
+
+
+## Quickstart
+
+Assume:
+1. `table` is a LanceDB Table
+2. `text` is the name of the Table column that we want to index
+
+To create the index:
+
+```python
+table.create_fts_index("text")
+```
+
+To search:
+
+```python
+df = table.search("puppy").limit(10).select(["text"]).to_df()
+```
+
+LanceDB automatically looks for an FTS index if the input is str.
```

Comment:
This is a python only feature, right?

Reply:
For now. Ideally we can integrate Tantivy from Rust but I think that takes more architectural clarity than we currently have (since we don't really know how the usage will look)

Reply:
Add some doc w.r.t. of this python only API?

Reply:
Yup will do